### ISSUE TITLE
Небольшое предложение по внешнему виду

### ISSUE THREAD

**Issue by AerialA** [kreshikhin/clumsy#4]
При анимации выглядело бы еще лучше если бы шрифт тоже преобразовывался каждый кадр. 
Наличие неизменящихся элементов в подобной стилистике выглядит не особо симпатично

**Comment by kreshikhin** [comment]
Да, согласен. Но тут надо отрисовывать текст офлайн и что-то с ним делать. Либо писать алгоритм которые забирает данны через context.getImageData(x, y, width, height) и медленно и грузно их обрабатывает добавляя искажения.
Либо тоже саме сделать через webgl, что было бы быстрее.

Второй путь конечно же исключает возможность использованися node-canvas.
